samples/azure_attestation/README.md

```diff
@@ -1,13 +1,13 @@
 # Go remote attestation sample using Microsoft Azure Attestation
-This sample shows how to do remote attestation of an EGo enclave in combination with [Microsoft Azure Attestation](https://docs.microsoft.com/en-us/azure/attestation/). It consists of a server running in the enclave and a client (the relying party) that attests the server before sending a secret. The Azure Attestation Provider simplifies the attestation process for the client.
+This sample shows how to do remote attestation of an EGo enclave in combination with [Microsoft Azure Attestation](https://docs.microsoft.com/en-us/azure/attestation/). It consists of a server running in an enclave and a client (the relying party) that attests the server before sending a secret. The Azure Attestation Provider simplifies the attestation process for the client.
 
-**Note: This sample only works on SGX-FLC systems. Some error handling in this sample is omitted for brevity.**
+**Note: This sample only works on SGX-FLC systems. Some error handling is omitted for brevity.**
 
 ## How it works
 
 ![azure attestation sample](illustration.svg)
```

Comment:
why did you delete illustration.drawio?

Reply:
Felix mentioned that the svg might be enough, so I tested a little and found that you can open the svg in drawio and edit it without problems, so the .drawio isn't needed.